### DISCUSSION TITLE
What ports do I have to open in the firewall for discovery mode?

### DISCUSSION THREAD

**Discussion by einarpersson** [Spotifyd/spotifyd#1235] [Q&A] [answered]
I am running on a home server with ubuntu, and `ufw` firewall.

When disabling `ufw` I can get discovery to work, but not otherwise.

**What ports do I have to open up to get discovery mode to work?**

**Comment by eladyn** [comment, marked as answer]
Ah, I saw your comment on the issue before this discussion, see [here](https://github.com/Spotifyd/spotifyd/issues/998#issuecomment-1807867264) for an answer.
